lib/community.ex

```diff
@@ -18,7 +18,17 @@ defmodule Community do
     # See http://elixir-lang.org/docs/stable/elixir/Supervisor.html
     # for other strategies and supported options
     opts = [strategy: :one_for_one, name: Community.Supervisor]
-    Supervisor.start_link(children, opts)
+    {:ok, pid} = Supervisor.start_link(children, opts)
+
+    seed_database
```

Comment:
How about adding this as a mix command instead of running it on startup? I know we done that on Flywheel but that's a special case where it has to be in the same process and I don't think that's necessary here.